$NetBSD: patch-configure,v 1.2 2013/01/29 15:02:24 tsutsui Exp $

- use ${EGDIR} for config file
- always use atomic_ops(3) on NetBSD, even on arm
- kill test(1) bash'ism

--- configure.orig	2012-07-19 11:30:11.000000000 +0000
+++ configure
@@ -7995,6 +7995,11 @@ else
     { $as_echo "$as_me:${as_lineno-$LINENO}: checking architecture for native atomic operations" >&5
 $as_echo_n "checking architecture for native atomic operations... " >&6; }
     case $host in
+        *-netbsd*)
+            { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
+$as_echo "yes" >&6; }
+            need_libatomic_ops=no
+        ;;
         arm*)
             { $as_echo "$as_me:${as_lineno-$LINENO}: result: arm" >&5
 $as_echo "arm" >&6; }
@@ -8062,11 +8067,6 @@ $as_echo "#define ATOMIC_ARM_INLINE_ASM 
 fi
             fi
         ;;
-        *-netbsdelf5*)
-            { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-            need_libatomic_ops=no
-        ;;
         *-freebsd*)
             { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
@@ -19748,7 +19748,7 @@ _ACEOF
 
 fi
 
-if test "$ac_cv_tls" == "__thread"; then :
+if test "$ac_cv_tls" = "__thread"; then :
 
 $as_echo "#define SUPPORT_TLS___THREAD 1" >>confdefs.h
 
@@ -20146,7 +20146,7 @@ ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 fi
 
-if test "x$ax_pthread_ok" == "xyes"; then :
+if test "x$ax_pthread_ok" = "xyes"; then :
 
 $as_echo "#define _POSIX_PTHREAD_SEMANTICS 1" >>confdefs.h
 
@@ -26138,7 +26138,7 @@ _ACEOF
 PA_SOEXT=.so
 
 
-pulseconfdir="${sysconfdir}/pulse"
+pulseconfdir="${EGDIR}"
 
 
   prefix_NONE=
